tests/ascii_data_boundary_velocity_residual_shell.prm

```diff
@@ -0,0 +1,97 @@
+##### simple test for ascii data
+
+set Dimension                              = 3
+
+set Use years in output instead of seconds = true
+set Output directory                       = ascii_data_boundary_velocity_residual_shell 
+set End time                               = 1e3
+
+set Adiabatic surface temperature          = 1613.0
+
+subsection Geometry model
+  set Model name = spherical shell
+
+  subsection Spherical shell
+    set Opening angle = 90
+    set Inner radius = 3481000
+    set Outer radius = 6336000
+  end
+end
+
+subsection Initial temperature model
+  set Model name = harmonic perturbation
+  subsection Harmonic perturbation
+    set Magnitude = 30
+    set Reference temperature = 1613
+  end
+end
+
+
+subsection Boundary temperature model
+  set List of model names = spherical constant
+  subsection Spherical constant
+    set Inner temperature = 3000
+    set Outer temperature = 273
+  end
+end
+
+
+# The parameters below this comment were created by the update script
+# as replacement for the old 'Model settings' subsection. They can be
+# safely merged with any existing subsections with the same name.
+
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators   = inner, outer
+end
+
+subsection Boundary velocity model
+  set Prescribed velocity boundary indicators = west:ascii data, east:ascii data, south:ascii data, inner:ascii data, outer: ascii data
+end
+
+
+subsection Boundary velocity model
+  subsection Ascii data model
+    set Data file name       = shell_3d_%s.0.txt
+    
+    set Data directory = $ASPECT_SOURCE_DIR/data/boundary-velocity/ascii-data/test/
+    set Scale factor = 1
+  end
+end
+
+
+subsection Gravity model
+  set Model name = radial constant
+
+  subsection Radial constant
+    set Magnitude = 10
+  end
+end
+
+
+subsection Material model
+  set Model name = simple
+  subsection Simple model
+    set Viscosity = 1e21
+  end
+end
+
+
+subsection Mesh refinement
+  set Initial global refinement                = 1
+  set Initial adaptive refinement              = 0
+  set Time steps between mesh refinement       = 0
+  set Strategy                                 = temperature
+end
+
+
+subsection Postprocess
+  set List of postprocessors = velocity statistics, temperature statistics, heat flux statistics, visualization
+  subsection  Visualization
+    set List of output variables      = adiabat, material properties, gravity, heat flux map, strain rate, boundary velocity residual
```

Comment:
please remove everything but `boundary velocity residual` to make the output files as small as possible. Also switch the output format to 'gnuplot'. The default .vtu output is compressed binary data, and that is hard to compare against reference results (as soon as one number changes only slightly the whole output will be different), but .gnuplot data is stored as ascii text.